packages/remix-dev/compiler/plugins/cssFilePlugin.ts

```diff
@@ -119,3 +129,33 @@ export function cssFilePlugin(options: {
     },
   };
 }
+
+function postcssPlugin({
+  postcssProcessor,
+  options,
+}: {
+  postcssProcessor: Processor;
+  options: CompileOptions;
+}): esbuild.Plugin {
+  return {
+    name: "postcss-plugin",
+    async setup(build) {
+      build.onLoad({ filter: /\.css$/, namespace: "file" }, async (args) => {
```

Comment:
I know we're going to look at Sass in a separate proposal, but is there any harm in allowing `.scss` files to get picked up by this plugin? We could catch potential errors from postcss processing and let users know they need to configure postcss to parse and transform Sass.

Reply:
I think there's a bit of effort involved for this to work since you need a different parser for `.scss` files, and I'd want to have tests for it too. I'd prefer to do this as part of the Sass integration to ensure everything's working together correctly.